tests/foreman/ui/test_rhcloud_insights.py

```diff
@@ -215,3 +217,221 @@ def test_host_sorting_based_on_recommendation_count():
 
     :CaseAutomation: ManualOnly
     """
+
+
+@pytest.mark.run_in_one_thread
+@pytest.mark.tier2
+def test_host_details_page(
+    rhel8_insights_vm,
+    fixable_rhel8_vm,
```

Comment:
Is the `fixable_rhel8_vm` used in this test?

Reply:
It's same as those commands I have for other test, only difference is it's a fixture which use `rhel8_insights_vm`.
Removed it and directly running the commands from test.